pace-util/tests/mpi/test_mpi_mock.py

```diff
@@ -301,7 +301,7 @@ def test_worker(comm, dummy_results, mpi_results, numpy):
         assert len(dummy_results) == len(mpi_results)
         for dummy, mpi in zip(dummy_results, mpi_results):
             if isinstance(mpi, numpy.ndarray):
-                numpy.testing.assert_array_equal(dummy, mpi)
+                numpy.testing.assert_array_equal(dummy.data, mpi.data)
```

Comment:
Double check that we can use `numpy.testing.assert_array_equal ` on `gt4py.storage.Storage` instances.

Reply:
@mcgibbon Yes, this is supported. Can reproduce with
```python
import numpy as np
from gt4py import storage
default_origin = (0, 0, 0)
shape = (6, 6, 3)
for backend in ("gtc:numpy", "gtc:gt:cpu_ifirst"):
    args = (backend, default_origin, shape, float)
    np.testing.assert_array_equal(storage.ones(*args), storage.ones(*args))
```